ldap/main.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_iam_user" "HR" {
+  name = "HR"
+}
+
+resource "aws_vpc" "ldap" {
+  cidr_block = "10.1.0.0/16"
+  instance_tenancy = "default"
+
+  tags = {
+    "terraform" = ""
+    "component" = "HR"
+  }
+}
+
+resource "aws_subnet" "ldap1" {
+  vpc_id            = aws_vpc.ldap.id
+  availability_zone = "sa-east-1a"
+  cidr_block        = "10.1.1.0/24"
+}
+
+resource "aws_subnet" "ldap2" {
+  vpc_id            = aws_vpc.ldap.id
+  availability_zone = "sa-east-1b"
+  cidr_block        = "10.1.2.0/24"
+}
+
+resource "aws_directory_service_directory" "ldap" {
+  name     = "corp.bbbfake.com"
+  password = "SuperSecretPassw0rd"
```

Comment:
Is it okay to have a password here? I'm not familiar with these settings, though.

Reply:
No, but this directory is not acessible yet. The plan is create with this password, use the aws console to create a admin user, configure ssl, change password, put the password on a github secret, then use the secret here instead of a plain password, and in the end allow connections from the internet.